vogue/commands/load/lims.py

```diff
@@ -1,16 +1,30 @@
 import click
+from vogue.load.sample import load_lims_sample, dry_run, load_all_samples
 
-from .single import single as single_command
-from .multi import multi as multi_command
 
-@click.group()
+@click.command("lims", short_help = "load lims into db.")
+@click.option('-s', '--sample-lims-id', help = 'Input sample lims id')
+@click.option('--load-all/--no-load-all', default = False, help = 'Loads all lims sample ids')
+@click.option('--dry/--no-dry', default = True, show_default = True,
```

Comment:
Why do you need to have two options for these? It is just a flag yes/no. If dry run is used on command line perform a dry run, else not.
```suggestion
@click.option('--dry-run', is_flag=True, help='If dry run')
```